tests/javascript-config/unit/jest.config.json

```diff
@@ -8,35 +8,28 @@
 		"!**/vendor/**",
 		"!**/test/**"
 	],
-	"modulePaths": [
-		"<rootDir>"
-	],
+	"modulePaths": ["<rootDir>"],
 	"moduleNameMapper": {
 		"@eventespresso\\/(eejs)": "assets/ZZZ/$1",
-		"@eventespresso\/core-blocks": "assets/ZZZ/blocks",
-		"@eventespresso\/components": "assets/ZZZ/components",
-		"@eventespresso\/data": "assets/ZZZ/data",
-		"@eventespresso\/helpers": "assets/ZZZ/data/helpers",
-		"@eventespresso\/model": "assets/ZZZ/data/model",
-		"@eventespresso\/validators": "assets/ZZZ/eejs/validators",
-		"@eventespresso\/editor-hocs": "assets/ZZZ/editor-hocs",
-		"@eventespresso\/editor": "assets/ZZZ/editor",
-		"@eventespresso\/hocs": "assets/ZZZ/higher-order-components",
-		"@eventespresso\/hooks": "assets/ZZZ/hooks",
-		"@eventespresso\/value-objects": "assets/ZZZ/vo",
-		"@eventespresso\/utils": "assets/ZZZ/utils",
```

Comment:
i wonder if this was added for Windows compatibility?

Reply:
dunno, I've opened this file and prettier formatted it on save

Reply:
will move these configurations to new ts-oriented jest config and will test there

Reply:
had reverted back these changes since rebasing with master had dismissed your approval